mlflow/tracking/client.py

```diff
@@ -200,6 +230,52 @@ class MlflowClient:
     can keep the implementation of the tracking and registry clients independent from each other.
     """
 
+    _image_logging_queue = None
+
+    class _ImageAsyncLoggingQueue:
```

Comment:
We don't need to implement a new Queue class,

let's reuse `AsyncArtifactsLoggingQueue`,
and update `log_artifact_handler` ( you need to add a `save_option` param for `log_artifact_handler`
https://github.com/mlflow/mlflow/blob/b88f9f6c54394de1b9c6279f55518c211b66897d/mlflow/store/artifact/artifact_repo.py#L97